src/lsst/cm/prod/configs/DC2/test/dc2_test-med-1.yaml

```diff
@@ -0,0 +1,131 @@
+prepare:
+    class_name: lsst.cm.tools.db.script_handler.PrepareScriptHandler
+ancil:
+    class_name: lsst.cm.tools.db.script_handler.AncillaryScriptHandler
+job:
+    class_name: lsst.cm.tools.db.job_handler.JobHandler
+workflow:
+    class_name: lsst.cm.tools.db.workflow_handler.WorkflowHandler
+    input_type: source
+    output_type: run
+    job_config: job
+rescue_workflow:
+    includes: ["workflow"]
+    rescue: True
+group:
+    class_name: lsst.cm.tools.db.group_handler.GroupHandler
+    input_type: source
+    output_type: run
+    workflow_config: workflow
+    templates:
+        coll_in: "{root_coll}/{production_name}/{campaign_name}_input"
+        coll_out: "{root_coll}/{production_name}/{campaign_name}_output"
+step:
+    class_name: lsst.cm.tools.db.step_handler.StepHandler
+    input_type: source
+    output_type: run
+    group_config: group
+    templates:
+        coll_out: "{root_coll}/{production_name}/{campaign_name}_output"
+step1:
+    prerequisites: []
+    includes: ["step"]
+    pipeline_yaml: "${DRP_PIPE_DIR}/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml#step1"
+    data_query_base: "instrument='LSSTCam-imSim' and skymap='DC2' and tract in (3828, 3829)"
+    split_args:
+        dataset: raw
+        field: exposure
+        min_queries: 4
+step2:
+    prerequisites: ["step1"]
+    includes: ["step"]
+    pipeline_yaml: "${DRP_PIPE_DIR}/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml#step2"
+    data_query_base: "instrument='LSSTCam-imSim' and skymap='DC2' and tract in (3828, 3829)"
+step3:
+    prerequisites: ["step2"]
+    includes: ["step"]
+    pipeline_yaml: "${DRP_PIPE_DIR}/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml#step3"
+    data_query_base: "instrument='LSSTCam-imSim' and skymap='DC2'"
+    split_vals:
+        field: tract
+        values:
+            - 3828
+            - 3829
+step4:
+    prerequisites: ["step3"]
+    includes: ["step"]
+    pipeline_yaml: "${DRP_PIPE_DIR}/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml#step4"
+    data_query_base: "instrument='LSSTCam-imSim' and skymap='DC2' and tract in (3828, 3829)"
+    split_args:
+        dataset: calexp # test or research the reasonable arguments here
+        field: visit
+        min_queries: 4
+step5:
+    prerequisites: ["step4"]
+    includes: ["step"]
+    pipeline_yaml: "${DRP_PIPE_DIR}/pipelines/LSSTCam-imSim/DRP-test-med-1.yaml#step5"
+    data_query_base: "instrument='LSSTCam-imSim' and skymap='DC2' and tract in (3828, 3829)"
```

Comment:
Don't think you want that `tract in (3828, 3829)` here

Reply:
Do you know if it will mess anything up?

Reply:
I'm not sure, but it is cleaner to get rid of it, since you are specifying per-tract groups right below

Reply:
During pair coding I made the argument to keep the tract constraints here.  The reasoning was that in step 4 there will be difference images that succeed that only have partial coverage from the tract 3828 and 3829 templates. (ie they're hanging off the tract)

There won't be any diaSources outside the tract boundaries, so in the end you'll still only end up with data in for the 3828, 3829. (the outputs should be the same either way), But my fear is that if you don't include the tract boundaries, the QG generator might ATTEMPT to associate diaSources outside the tract boundaries and waste some query time and init time trying too. 

I haven't tried it though.